mod/profiles.php

```diff
@@ -473,11 +465,13 @@ function profiles_post(App $a) {
 			intval(local_user())
 		);
 
-		if($r)
-			info( t('Profile updated.') . EOL);
+		/// @TODO decide to use dbm::is_result() here and check $r
```

Comment:
`if($r)` is found 12 times scattered over several files. I have it already in my todos.txt and fix it next round.